pkg/sqlqueries/SQL/mto/category/create.sql

```diff
@@ -3,20 +3,34 @@ INSERT INTO mto_category (
     name,
     parent_id,
     model_plan_id,
+    position,
     created_by
 )
 VALUES (
     :id,
     :name,
     :parent_id,
     :model_plan_id,
+    COALESCE(
+        ( -- Place category at the bottom in order
+        -- TODO also join on model plan ID!!!! Otherwise the order is for everything
```

Comment:
Guessing we can nix this second comment, as I see the `WHERE` statement captures the `model_plan_id` part now

Reply:
Good call! Thanks Clay :D